pacman-nodejs-app/tools/migrate/pacman.sh

```diff
@@ -2,15 +2,32 @@
 set -e
 
 function add_new_mongo_instance {
-    MONGO_SRC_POD=$(kubectl --context ${SRC_CONTEXT} get pod \
-    --selector="name=mongo" --output=jsonpath='{.items..metadata.name}')
+    
+	MONGO_SRC_POD=$(kubectl --context ${SRC_CONTEXT} get pod \
+		--selector="name=mongo" \
+		--output=jsonpath='{.items..metadata.name}')
+	
+	MONGO_DST_POD=$(kubectl --context ${DST_CONTEXT} get pod \
+        	--selector="name=mongo" \
+		--output=jsonpath='{.items..metadata.name}')
+	
+	kubectl --context ${SRC_CONTEXT} exec -it ${MONGO_SRC_POD} -- \
```

Comment:
This is really only necessary if the destination mongo host is a FQDN and not an IP. It would help speed up the migration when not using DNS names but it's not too big a deal.